core/notifications/src/job/mod.rs

```diff
@@ -196,6 +243,29 @@ pub async fn spawn_multi_user_event_dispatch(
     Ok(())
 }
 
+#[instrument(name = "job.spawn_link_email_reminder", skip_all, fields(error, error.level, error.message), err)]
+pub async fn spawn_link_email_reminder(
+    pool: &sqlx::PgPool,
+    data: impl Into<LinkEmailReminderData>,
+) -> Result<(), JobError> {
+    let data = data.into();
+    // TODO: I think reusing the id here might cause a bug
+    match JobBuilder::new_with_id(LINK_EMAIL_REMINDER_ID, "link_email_reminder")
```

Comment:
Since we are recursively calling back into this for pagination I think this may be a bug @thevaibhav-dixit @bodymindarts